spec/factories/products.rb

```diff
@@ -85,6 +85,12 @@
       create :instrument_price_policy, price_group: product.facility.price_groups.last, usage_rate: 1, product: product
       product.reload
     end
+
+    trait :reservation_only do
+      after(:create) do |inst, evaluator|
+        inst.relay = FactoryGirl.create(:relay, :manual_control, :instrument => inst)
```

Comment:
reservation_only should not have a relay. control_mechanism should just be nil (manual_control).